runtime/autoload/fcnotify.vim

```diff
@@ -0,0 +1,46 @@
+if exists('g:loaded_watcher_provider')
+  finish
+endif
+
+let s:save_cpo = &cpo " save user coptions
+set cpo&vim " reset them to vim defaults
+
+command! -nargs=1 Watch call v:lua.vim.fcnotify.start_watch(expand('<args>'))
+command! -nargs=1 Stop call v:lua.vim.fcnotify.start_watch(expand('<args>'))
```

Comment:
`Stop` ifs too generic, both commands should feel similar: WatchStart / WatchStop for instance or

Reply:
Removed the commands